regolith/helpers/a_proposalhelper.py

```diff
@@ -0,0 +1,197 @@
+"""Helper for adding a proposal to the proposals.yml collection.
+"""
+import datetime as dt
+import dateutil.parser as date_parser
+from dateutil.relativedelta import relativedelta
+import sys
+
+from regolith.helpers.basehelper import DbHelperBase
+from regolith.fsclient import _id_key
+from regolith.tools import (
+    all_docs_from_collection,
+    get_pi_id,
+    fuzzy_retrieval
+)
+
+TARGET_COLL = "proposals"
+
+def subparser(subpi):
+    # Do not delete --database arg
+    subpi.add_argument("--database",
+                       help="The database that will be updated.  Defaults to "
+                            "first database in the regolithrc.json file."
+                       )
+    subpi.add_argument("name", help="A short but unique name for the proposal",
+                       default=None
+                       )
+    subpi.add_argument("amount", help="value of award",
```

Comment:
these are required args, so I don't think they should take defaults, at least I can't see how they would ever take the default value since the code won't run if they are not specified.